Concepts/concept_reference_term_type.md

```diff
@@ -0,0 +1,104 @@
+# Concepts Reference Term Type
+
+## Overview
+
+* A concept term defines a medical coding term or OpenMRS concept dictionary term that could be mapped to a concept. 
+A medical coding term is used to represent a name from a standard medical code (ie.  SNOMED, ICD10, LOINC, etc). OpenMRS 
+concept dictionary terms are used to refer to concepts by concept_id or name of a concept in the concept dictionary (ie. CIEL, PIH, AMPATH, etc.).  
+  
+## Available operations. 
+1.  [List concept reference term types](#list-concept-reference-term-types)
+2.  [Create a concept reference term type](#create-a-concept-reference-term-type)
+3.  [Update a concept reference term type](#update-a-concept-reference-term-type)
+4.  [Delete a concept reference term type](#delete-a-concept-reference-term-type)
+
+### List concept reference term types
+
+* #### List all concepts reference term types.
+
+    Quickly filter concept reference term types with given query parameters.Returns a `404 Not Found` status if concept 
+    reference term type not exists. If user not logged in to perform this action,a `401 Unauthorized` status returned.
+
+    ##### Query Parameters
+
+    Parameter | Type | Description
+    --- | --- | ---
+    *q* | `Search Query` | Display Name of oncepts reference term type object
```

Comment:
Does this work for you? In my testing, I was unable to find any concept reference terms using the `q` parameter.